docs/README.md

```diff
@@ -231,11 +231,13 @@ completely deleted in the PR.
        message: 'Custom message...' # this is optional, a default message is used when not specified.
     must_include:
        regex: '### Goals|### Changes'
+       regex_flag: '' # Optional. Specify the flag for Regex. default is 'i', to disable default use ''
```

Comment:
I would personally prefer to specify `none` over `''` to override the default. I feel it is more intuitive. Thoughts?

Reply:
hmm... I agree. putting `''` felt a little off to me as well.